packages/ava/ava.config.cjs

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  timeout: '60000',
```

Comment:
Since AVA’s timeout works differently than Jest’s, can we make this smaller?

Reply:
Sure is 30s good?

Reply:
It’s going to be the timeout for _each RPC request_, right? 10s seems more than sufficient, if so. Even for testnet.